src/main/java/com/beerjournal/breweriana/item/rating/persistence/Rating.java

```diff
@@ -0,0 +1,28 @@
+package com.beerjournal.breweriana.item.rating.persistence;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@Document
+@Data
+@EqualsAndHashCode(exclude = "id")
+@RequiredArgsConstructor(access = PRIVATE)
+public final class Rating {
+    @Id
+    private final ObjectId id;
```

Comment:
Just by a need (or necessity) to use embedded collection approach.  
All the time I had in mind that these ratings in the future won't consist only of one int value, but there will be more data (comment, date, whatever more...).  
Also - items and their ratings are now loosely coupled. We can even call them separate _components_ :D  
Then I will leave this model now as it is.